src/components/models/PryvServiceExtension.js

```diff
@@ -242,3 +242,25 @@ Pryv.Service.prototype.changePassword = async function changePassword (
     });
   return res.status;
 };
+
+Pryv.Service.prototype.loginWithThrow = async function loginWithThrow (username, password, appId, originHeader) {
+  const apiEndpoint = await this.apiEndpointFor(username);
+
+  const headers = { accept: 'json' };
+  originHeader = originHeader || (await this.info()).register;
```

Comment:
For origin header in lib-js there is a check  (!utils.isBrowser()). And in general why you put it here and not in lib-js?